palantir-java-format/src/test/resources/com/palantir/javaformat/java/testdata/B20701054.output

```diff
@@ -14,29 +14,16 @@ class B20701054 {
         ImmutableList<String> x = ImmutableList.INSTANCE.add(1).build();
         ImmutableList<String> x = ImmutableList.INSTANCE.add(1).add(2).build();
         ImmutableList<String> x = ImmutableList.INSTANCE.add(1).add(2).add(3).build();
-        ImmutableList<String> x = ImmutableList.INSTANCE
-                .add(1)
-                .add(2)
-                .add(3)
-                .add(4)
-                .build();
+        ImmutableList<String> x =
+                ImmutableList.INSTANCE.add(1).add(2).add(3).add(4).build();
 
         ImmutableList<String> x = ImmutableList.builder().add(1).build();
         ImmutableList<String> x = ImmutableList.builder().add(1).add(2).build();
         ImmutableList<String> x = ImmutableList.builder().add(1).add(2).add(3).build();
-        ImmutableList<String> x = ImmutableList.builder()
-                .add(1)
-                .add(2)
-                .add(3)
-                .add(4)
-                .build();
-        ImmutableList<String> x = ImmutableList.builder()
-                .add(1)
-                .add(2)
-                .add(3)
-                .add(4)
-                .add(5)
-                .build();
+        ImmutableList<String> x =
+                ImmutableList.builder().add(1).add(2).add(3).add(4).build();
+        ImmutableList<String> x =
+                ImmutableList.builder().add(1).add(2).add(3).add(4).add(5).build();
```

Comment:
Are these examples really better? It's harder to scan chained calls horizontally than when they're lined up vertically.

Reply:
(apparently this keeps it consistent with other parts of the formatter, rather than letting them fight)